design/account.go

```diff
@@ -109,6 +130,24 @@ var _ = a.Resource("users", func() {
 		a.Response(d.BadRequest, JSONAPIErrors)
 	})
 
+	a.Action("createUserAsServiceAccount", func() {
```

Comment:
We don't have any REST API for creating users yet. Why do not call it Create (it will be still allowed to the service account only though). Later we will use our AuthZ service to determine who can do what.

Reply:
sounds right, thanks.
a076d63